### ISSUE TITLE
[CLOSED] Add lightning rounds, strategic play combat option

### ISSUE THREAD

**Issue by toddmedema** [ExpeditionRPG/expedition#179]
<a href="https://github.com/smartin015"><img src="https://avatars2.githubusercontent.com/u/607666?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [smartin015](https://github.com/smartin015)**
_Sunday May 08, 2016 at 16:52 GMT_
_Originally opened as https://github.com/ExpeditionRPG/expedition-app/issues/40 (3 comment(s))_

----

Round is a fixed amount of time. If you go over the time, you take a bonus damage amount (up to enemy tier?)

When the round is done, generate a random spread of damage numbers that correspond to a dice roll. 
1-5: 0 damage
5-12: 1 damage
13-17: 2 damage 
18-20: 3 damage

**Comment by toddmedema** [comment]
<a href="https://github.com/smartin015"><img src="https://avatars2.githubusercontent.com/u/607666?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smartin015](https://github.com/smartin015)**
_Sunday May 08, 2016 at 16:52 GMT_

----

See Asana for whom to notify once this is complete.

**Comment by toddmedema** [comment]
<a href="https://github.com/smartin015"><img src="https://avatars2.githubusercontent.com/u/607666?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [smartin015](https://github.com/smartin015)**
_Sunday May 08, 2016 at 17:00 GMT_

----

Similar to another task: "App combat: 3 different difficulties, easy, normal, hard"

**Comment by toddmedema** [comment]
<a href="https://github.com/toddmedema"><img src="https://avatars2.githubusercontent.com/u/775657?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [toddmedema](https://github.com/toddmedema)**
_Saturday Jan 07, 2017 at 00:27 GMT_

----

Closing, duplicate of https://github.com/Fabricate-IO/expedition-app/issues/124
